src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib

```diff
@@ -249,6 +249,32 @@
             <point key="canvasLocation" x="-442" y="78"/>
         </view>
     </objects>
+    <designables>
+        <designable name="7CF-il-Sk2">
+            <size key="intrinsicContentSize" width="41.5" height="20.5"/>
+        </designable>
+        <designable name="9f7-Oe-AOV">
+            <size key="intrinsicContentSize" width="67" height="50"/>
+        </designable>
+        <designable name="Nk7-Zk-JE2">
+            <size key="intrinsicContentSize" width="70.5" height="20.5"/>
+        </designable>
+        <designable name="PuR-3M-Pvl">
+            <size key="intrinsicContentSize" width="77" height="33.5"/>
+        </designable>
+        <designable name="lge-Dm-wcC">
+            <size key="intrinsicContentSize" width="41.5" height="20.5"/>
+        </designable>
+        <designable name="vh2-Dv-3W6">
+            <size key="intrinsicContentSize" width="33" height="16"/>
+        </designable>
+        <designable name="vo8-Zn-enn">
+            <size key="intrinsicContentSize" width="54" height="50"/>
+        </designable>
+        <designable name="znr-XX-rfp">
+            <size key="intrinsicContentSize" width="85" height="18"/>
+        </designable>
+    </designables>
```

Comment:
Why are these changed as well, it is required for the fix?

Reply:
thanks for your reply ! no these changed by when I opened xib file of OnboardingViewController.xib , not required for the fix @KaiTeuber